common/file/archive/zip_test.go

```diff
@@ -33,60 +36,39 @@ func TestUnZip(t *testing.T) {
 
 func TestZip(t *testing.T) {
 	tempDir := t.TempDir()
-	singleFile := filepath.Join("..", "..", "..", "testdata", "configtest.json")
 	outFile := filepath.Join(tempDir, "out.zip")
-	err := Zip(singleFile, outFile)
-	if err != nil {
-		t.Fatal(err)
-	}
+	err := Zip(filepath.Join("..", "..", "..", "testdata", "configtest.json"), outFile)
+	require.NoError(t, err, "Zip should not error")
 	o, err := UnZip(outFile, tempDir)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if len(o) != 1 {
-		t.Fatalf("expected 1 files to be extracted received: %v ", len(o))
-	}
+	require.NoError(t, err, "UnZip should not error")
+	assert.Len(t, o, 1, "Should extract 1 file")
 
-	folder := filepath.Join("..", "..", "..", "testdata", "http_mock")
+	folder := filepath.Join("..", "..", "..", "testdata", "gctscript")
 	outFolderZip := filepath.Join(tempDir, "out_folder.zip")
 	err = Zip(folder, outFolderZip)
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err, "Zip should not error")
 	o, err = UnZip(outFolderZip, tempDir)
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err, "UnZip should not error")
 	var found bool
 	for i := range o {
-		if filepath.Base(o[i]) == "binance.json" {
+		if filepath.Base(o[i]) == "timer.gct" {
 			found = true
 		}
 	}
-	if !found {
-		t.Fatal("could not find file in zip")
-	}
-
-	if expected := 6; len(o) < expected {
-		t.Fatalf("expected at least %v files to be extracted, received: %v ", expected, len(o))
-	}
+	assert.True(t, found, "Should find a gctscript in the zip")
+	assert.Equal(t, 9, len(o), "Should extract 6 files")
 
 	folder = filepath.Join("..", "..", "..", "testdata", "invalid_file.json")
-	outFolderZip = filepath.Join(tempDir, "invalid.zip")
-	err = Zip(folder, outFolderZip)
-	if err == nil {
-		t.Fatal("expected IsNotExistError on invalid file")
-	}
+	err = Zip(folder, filepath.Join(tempDir, "invalid.zip"))
+	assert.ErrorContains(t, err, "no such file or directory", "Zip should error correctly")
```

Comment:
```suggestion
	assert.ErrorIs(t, err, fs.ErrNotExist, "Zip should error with fs.ErrNotExist")
```

This is cross platform error friendly as Billy G's tests are currently unhappy

Reply:
Thanks. I went hunting for what to use and didn't quickly find it.

Fixed  6b59bda0